core/src/test/resources/configs/my-custom-config.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Happy with configs but I wonder if the name of the files could be something more meaningful. As a reader, you might wonder what might be the difference between `my-custom-config23.xml` and `my-custom-config534.xml` ;)